Directory.Packages.props

```diff
@@ -16,6 +14,8 @@
     <PackageVersion Include="Fork.Coypu" Version="3.4.0" />
     <PackageVersion Include="JetBrains.Annotations" Version="2023.3.0" />
     <PackageVersion Include="log4net" Version="2.0.17" />
+    <PackageVersion Include="Microsoft.Build.Framework" Version="17.8.3" />
+    <PackageVersion Include="Microsoft.Build.Utilities.Core" Version="17.8.3" />
```

Comment:
I'm not sure, I thought this fixed some errors during the changes, but removing it now changed nothing.

Reply:
I found the reason later on but forgot to track back to here: it was added after switching Security.MSBUild.Tasks project from Framework to .NET 8. Since we have now deleted that project, it's all good and we don't need the lines any more.